src/options/options.html

```diff
@@ -14,6 +14,10 @@
                 <label>Open as popup:</label>
                 <input id="gitpod-open-as-popup" type="checkbox"/>
             </div>
+            <div class="row">
+                <label>Replace <kbd>.</kbd> keybind (opens Gitpod instead of github.dev):</label>
```

Comment:
**suggestion**: Instead of using an aggresive replacement term here, what do you think of simply a) describing the keybinding set and b) skipping the explicit reference to `github.dev`.

```suggestion
                <label>Enable <kbd>.</kbd> keybind to open repositories in Gitpod</label>
```

Reply:
Good idea, I'm currently investigating how we can rewrite the keybind in GitLab, this will be really cool 😄